database/engine/rrdenginelib.c

```diff
@@ -1,25 +1,49 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 #include "rrdengine.h"
 
-void print_page_cache_descr(struct rrdeng_page_cache_descr *page_cache_descr)
+/* Caller must hold descriptor lock */
+void print_page_cache_descr(struct rrdeng_page_descr *descr)
 {
+    struct page_cache_descr *pg_cache_descr = descr->pg_cache_descr;
     char uuid_str[37];
     char str[512];
```

Comment:
More magic numbers, i see a printf below that has that 512 in order to fill str

Reply:
Yeah I should add this 512 to a define.